File: Iniciante/1080.cpp
#include <iostream>
using namespace std;

int main() {
    int n, max = 0, c = 0;
    for(int i = 1; i <= 100; ++i) {
        cin >> n;
        if(n >= max){
            c = i;
            max = n;
        }
    }
    cout << max << endl;
    cout << c << endl;
    return 0;
}

File: Iniciante/C#/1080.cs
using System;
class URI {
    static void Main(string[] args) { 
        int max = 0, c = 0;
        for(int i = 1; i <= 100; i++) {
            int n = int.Parse(Console.ReadLine());
            if(n >= max) {
                c = i;
                max = n;
            }
            
        }
        Console.WriteLine(max);
        Console.WriteLine(c);
    }   
}